.github/workflows/depcheck.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: |
           unset HAS_SECRET
           if [ -n "$GITHUB_TOKEN" ]; then HAS_SECRET=true; fi
-          echo ::set-env name=HAS_SECRET::${HAS_SECRET}
+          echo "name=HAS_SECRET" >> $GITHUB_ENV
```

Comment:
Thanks! This'll work for now, but suspect it will fail when someone on their own fork submits a PR. I think set-output is the replacement for set-env: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter

Reply:
No; This was the solution explicitly recommended in the deprecation announcement:

https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/